Search/Metadata/FieldEvaluator.php

```diff
@@ -72,6 +72,10 @@ public function getValue($object, FieldInterface $field)
 
     public function evaluateCondition($object, string $condition)
     {
+        if (!\is_array($object)) {
```

Comment:
why the object should be an array? what is the issue you try to solve here?

Reply:
the function can be called with any type and the expression language requires an array - so returning false